app/main/views.py

```diff
@@ -11,13 +11,18 @@ def index():
 
 @main.route('/get-resources')
 def get_resources():
-    resources = Resource.query.all()
+    resources = Resource.query.all() 
```

Comment:
nit: remove trailing whitespace please :) (if you're using Sublime, there is an extension that takes care of it for you)